app/models/form/sales/pages/buyer2_previous_housing_situation.rb

```diff
@@ -0,0 +1,11 @@
+class Form::Sales::Pages::Buyer2PreviousHousingSituation < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "buyer_2_previous_housing_situation"
+    @depends_on = [{ "buyer_two_not_already_living_in?" => true }]
```

Comment:
`buyer_two_not_already_living_in` sounds like they're living in the new property, but from what I'm gathering it's buyers lived together before this sale? Fine if it makes sense to you, but I didn't quite get the meaning of the method without having to go and look what it actually checks

Reply:
We should ask about buyer 2's previous housing situation if we do not know it, which is if buyer 2 is not already living in the house that is being bought.
I didn't find it obvious what to call this method, so I'm open to looking for a better name, considering it has failed the acid test of my fellow dev being able to easily read what it means.
I considered `buyer_two_already_living_in => false` (`buy2living == 1`), but I believe that if `buy2living => nil` and they navigate to the check answers page they would then see a question they shouldn't

Reply:
"At the time of purchase, was buyer 2 living at the same address as buyer 1?" `buyer_two_lived_at_the_same_address?` maybe?

Reply:
hello. This has already changed a bit after chatting with James to be `buyer_two_not_already_living_in_property?` what do you think about that?
As per previous comment, unless you disagree, I think that the method name has to be phrased in the negative, which is ever so slightly more awkward.
If you don't like `buyer_two_not_already_living_in_property?` then it could be `buyer_two_did_not_live_at_the_same_address?`, I'm fine with that if you prefer it, I find them equivalent.

Reply:
I don't mind what it it right now 👍